src/pages/CourseFeedback/components/HorizontalLabeledSlider.jsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Slider, Typography } from "@mui/material";
+import PropTypes from "prop-types";
+/**
+ * A custom tab panel component.
+ *
+ * @param {number} props.value - The value of the Slider component.
+ * @param {number} props.max - The maximum value of the Slider component.
+ * @param {string} props.label - The label for the Slider component.
+ * @param {function} props.onChange - The function to call when the Slider component changes.
+ * @returns {JSX.Element} - The HorizontalLabeledSlider component.
+ */
+function HorizontalLabeledSlider(props) {
+  return (
+    <Box>
+      <Typography variant="h6">{props.label}</Typography>
+      <Slider
+        size="medium"
+        color="primary"
+        orientation="horizontal"
+        value={props.value || 0}
+        valueLabelDisplay="auto"
+        marks
+        max={props.max}
```

Comment:
Does it make sense here to provide a default value as well?

Reply:
I thought it would be good, to guarantee that the sliders are at the starting position on render

Reply:
Yes, i meant a default value for the max props

Reply:
It probably doesn't hurt. Maybe a default of 10?

Reply:
Sounds good.
After that please merge the main into your branch and resolve conflicts